docs/headless-cms/08-01-add-new-documentation-to-console.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Add new documentation to the Console's documentation view
+type: Tutorial
+---
+
+This tutorial shows how you can customize the Documentation view that is available in the Console UI under the ![](./assets/docs.png) icon on the top navigation panel. The purpose of this tutorial is to create a new **Prometheus** documentation section that contains **Concepts** and **Guides** documentation topics with a set of Markdown subdocuments. The Markdown sources used in this tutorial point to specific topics in the official Prometheus documentation.  
+
+When you prepare ClusterDocsTopics, make sure that:
+- ClusterDocsTopics have a proper [custom resource definition](#custom-resource-clusterdocstopic).
+- Documents start with metadata, including obligatory `Title` for the document name and optional `Type` for any additional document grouping.
```

Comment:
Metadata names/keys should be lowercase otherwise it is confusing

Reply:
I removed that part